game/games/testgame/README.md

```diff
@@ -0,0 +1,16 @@
+# How to add you own Minigame - A Step by step Guide
+
+## 1. Chose a folder
+All files related only to your game go into a folder inside `res://games/`. You shouldn't rename it after the game is first merged. 
+There are no rules what to call it but a good idea is to use your name and the games name (ex. "asecondguy_assimilator") to avoid any accidental collisions.
```

Comment:
I agree, it avoids conflicts, but i don't think the initial author should be part of the game name.
I'd propose we change that requirement anyway, because the name of the game should not be that important, but rather use a unique ID (e.g. uuid) in the game.cfg. Makes it more portable as well.

Collisions are then also very unlikely to ever occur and we can then fix that through an update to the corresponding uuid.

Further suggestion might be to automatically generate a list of authors through a git blame / git log of the corresponding folder. Alternatively manually a'la https://allcontributors.org/

Reply:
Changing that is for another PR I think.